internal/photoprism/mediafile.go

```diff
@@ -378,6 +378,28 @@ func (m *MediaFile) ExtractEmbeddedVideo() (string, error) {
 
 			return dstPath, nil
 		}
+	} else if metaData.Error == nil && metaData.EmbeddedVideoOffset > 0 {
+		dstPath := filepath.Join(Config().SidecarPath(), m.RootRelPath(), m.BasePrefix(false)+".MP.mp4")
```

Comment:
Instead of hardcoding the `dstPath` extension, see the approach we followed here for Samsung motion photos: https://github.com/photoprism/photoprism/pull/3722/files#diff-e5a39f92547299cd358543ff78e747595e351aa0f56a50d43c0034cb94465d8aR368

Reply:
ok,  I can look to add the mime type check if we pick to go this way. Out of curiosity do we have any examples where its not actually a mp4? how critical is the check/logic and file move